components/devfile-page/Projects.tsx

```diff
@@ -75,21 +109,44 @@ interface Props {
  */
 
 const DevPageProjects = ({ starterProjects }: Props) => {
```

Comment:
Can you change the const name to `projects`

Reply:
resolved